packages/bun-types/package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "homepage": "https://bun.sh",
   "dependencies": {
-    "@types/node": "~20.12.8",
+    "@types/node": "~20.16.5",
```

Comment:
I was being conservative because I wasn't really sure how the version was chosen or if `bun` was aiming for compatibility with a particular version of node.
The version was last updated in 2a401486d9fc38b3795571f6a69b3e17afba91a9 and I didn't see why that specific version was chosen (I don't _think_ it was the latest on May 1st?). A specific version of `@types/node` (rather than `*`) was added in #8195 but looking through that I didn't see any reasoning behind how the version was chosen.
TL;DR - I'm happy to make this change (and the corresponding change in the lockfile) but would love to understand the more about the why of this version!

Reply:
Very reasonable. I'm curious as well. Thanks for pushing this along!